cmd/commands/runtime.go

```diff
@@ -783,6 +785,14 @@ func preInstallationChecks(ctx context.Context, opts *RuntimeInstallOptions) err
 		return fmt.Errorf("existing runtime check failed: %w", err)
 	}
 
+	if !opts.SkipNetworkTest {
+		err = util.TestNetwork(ctx, opts.KubeFactory, cfConfig.GetCurrentContext().URL)
+		handleCliStep(reporter.InstallStepRunPreCheckTestNetwork, "Testing the network", err, false)
```

Comment:
I would change everywhere from `TestNetwork` to `RunNetworkTest`. `InstallStepRunPreCheckTestNetwork` to `InstallStepRunPreCheckNetworkTest`. `Testing the network` to `Running network tests`. This is to be conformant with the rest of the steps.